share/functions/__fish_complete_man.fish

```diff
@@ -64,6 +64,9 @@ function __fish_complete_man
 		  print name, sect
 		}
 		'
+        
+        # Fish commands are not given by apropos
+        ls (dirname $__fish_datadir)/fish/man/man1/ ^/dev/null | string replace -r '\.1$' '\tFish command' | string match -r '^'$token'.*'
```

Comment:
Please don't use ls this way. The "dirname" is also useless - $__fish_datadir _is_ a directory, and is under our control.

Instead make the `string replace` a bit more complicated - `string replace -r '.*/([^/]+)\.1$' '$1\tFish command' -- $files`, where $files is `set -l files $__fish_datadir/fish/man/man1/*.1`.

(At least I think that's how the ".1" should be used. I think fedora gzips our man pages, so the regex potentially would have to handle "1.gz" as well)

Matching the token here should also be superfluous - we do that automatically.